docs/markdown/Getting Started/getting-started/prerequisites.md

```diff
@@ -42,9 +42,7 @@ System-specific notes
 
 > 📘 Apple Silicon (M1/M2) support
 >
-> If you have Python code, you may need to [set your interpreter constraints](doc:python-interpreter-compatibility) to Python 3.9+, as many tools, such as Black, will not install correctly when using earlier Python versions.
->
-> When running in Docker you will need to set `--no-watch-filesystem --no-pantsd`. (Although we don't recommend permanently setting this, as these options are crucial for performance when iterating.)
+> If running Pants inside a Docker container on an Apple Silicon machine you will need to set `--no-watch-filesystem --no-pantsd`. (Although we don't recommend permanently setting this, as these options are crucial for performance when iterating.)
```

Comment:
is it possible to link to why these need to be set specifically in containers only on apple silicon machines?

Reply:
@kaos do you know why this is?

Reply:
this would be due to the fact that the docker daemon is running in a linux virtual machine on Mac and does not provide proper support for the file notification system api:s across from the mounted files on the native Mac system over to the virtualized linux system.

Reply:
ah gotcha, seems worth adding the above context somewhere

Reply:
> Although we don't recommend permanently setting this, as these options are crucial for performance when iterating

Apple silicon users would need to permanently set this since they'd always use the container right?

Reply:
I think that it might be off topic for this PR, but nowadays, we might recommend using a `docker_environment` rather than running the entire build inside of Docker. Possibly premature though, as that's still experimental.

Reply:
> Apple silicon users would need to permanently set this since they'd always use the container right?

Well, this is only if you're running Pants inside a container. In normal use you'd run Pants natively directly on the machine.